examples/allennlp/allennlp_jsonnet.py

```diff
@@ -24,13 +24,16 @@
 import shutil
 
 import optuna
+from optuna.integration.allennlp import dump_best_config
 from optuna.integration import AllenNLPExecutor
 
 
 # This path trick is used since this example is also
 # run from the root of this repository by CI.
 EXAMPLE_DIR = os.path.dirname(os.path.abspath(__file__))
+CONFIG_PATH = os.path.join(EXAMPLE_DIR, "classifier.jsonnet")
 MODEL_DIR = "result"
+BEST_CONFIG_PATH = "best.classifier.json"
```

Comment:
I think it's a better practice not to include punctuations in the file besides for separating between file extensions.

Reply:
Thank you for pointing out. I fixed it in 5ab58d9.